docs/components/TokenColorTable.tsx

```diff
@@ -157,21 +156,25 @@ function ColorSquare({ color }: { color?: string }) {
 }
 
 export const ColorTable = ({
+	'aria-labelledby': ariaLabelledby,
+	'aria-describedby': ariaDescribedby,
 	tokens,
-	caption,
 }: {
+	'aria-labelledby': string;
+	'aria-describedby': string;
 	tokens: {
 		name: string;
 		desc: string;
 		color: string | undefined;
 	}[];
-	caption: string;
 }) => {
 	return (
 		<div className={proseBlockClassname}>
 			<TableWrapper>
-				<Table>
-					<TableCaption>{caption}</TableCaption>
+				<Table
```

Comment:
Rather than saying "Border colours" as the H2 and then "All border colours as the table caption a we can simply use H2 as the table description which removes a lot of duplicated content.